### PR TITLE
Makes changelog for MISKDJS up to date.

### PR DESCRIPTION
**Pull request by ammapspeople**

### CODE REVIEW

**Comment by marksdk** [comment]
@ammapspeople This is still open. Should we review or close?

**Comment by ammapspeople** [comment]
I see I forgot to add a reviewer, then forgot everything about it.

File: collections/_web/v3/CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,43 @@ nav_weight: 1000
...
+### Fixed
+- Fixed a bug where user roles were not passed along to distance matrix call.
+- Fixed bug where getBuilding returned undefined until the map had been moved.
+
+## [3.4.1] - 2019-10-30
+### Added
+- Client side caching. Most request are now beeing cached in the browser.
+- Added support for separate visibility control of label and icon
+
+### Fixed
```
Comment (marksdk) [inline collections/_web/v3/CHANGELOG.md]:
```suggestion
### Fixed

```

File: collections/_web/v3/CHANGELOG.md [hunk 2]
```diff
@@ -8,6 +8,43 @@ nav_weight: 1000
...
+- The DirectionsService will now return a route object with the status "ZERO_RESULTS" if no route could be calculated.
```
Comment (marksdk) [inline collections/_web/v3/CHANGELOG.md]:
```suggestion
- The `DirectionsService` will now return a route object with the status `ZERO_RESULTS` if no route could be calculated. Previously it threw an error.
```

File: collections/_web/v3/CHANGELOG.md [hunk 3]
```diff
@@ -8,6 +8,43 @@ nav_weight: 1000
...
+  Before it unintentional threw an error.
```
Comment (marksdk) [inline collections/_web/v3/CHANGELOG.md]:
```suggestion
```

File: collections/_web/v3/CHANGELOG.md [hunk 4]
```diff
@@ -8,6 +8,43 @@ nav_weight: 1000
...
+- Fixed a bug where the imageURL property on locations sometimes would be named imageurl.
```
Comment (marksdk) [inline collections/_web/v3/CHANGELOG.md]:
```suggestion
- Fixed a bug where the `imageURL` property on Locations sometimes would be named `imageurl`.
```

File: collections/_web/v3/CHANGELOG.md [hunk 5]
```diff
@@ -8,6 +8,43 @@ nav_weight: 1000
...
+
+## [3.4.0] - 2019-08-28
```
Comment (marksdk) [inline collections/_web/v3/CHANGELOG.md]:
```suggestion
## [3.4.0] - 2019-08-28

```

File: collections/_web/v3/CHANGELOG.md [hunk 6]
```diff
@@ -8,6 +8,43 @@ nav_weight: 1000
...
+### Changed
```
Comment (marksdk) [inline collections/_web/v3/CHANGELOG.md]:
```suggestion
### Changed

```

File: collections/_web/v3/CHANGELOG.md [hunk 7]
```diff
@@ -8,6 +8,43 @@ nav_weight: 1000
...
+- Floor calculation when building comes into focus according to agreed upon strategy, including respecting default floor of a building.
```
Comment (marksdk) [inline collections/_web/v3/CHANGELOG.md]:
```suggestion
- Floor calculation when Building comes into focus according to agreed upon strategy, including respecting default Floor of a Building.
```

File: collections/_web/v3/CHANGELOG.md [hunk 8]
```diff
@@ -8,6 +8,43 @@ nav_weight: 1000
...
+- The filter method now respects the fitView boolean also when clearing filter.
```
Comment (marksdk) [inline collections/_web/v3/CHANGELOG.md]:
```suggestion
- The `filter` method now respects the `fitView` boolean also when clearing filter.
```

File: collections/_web/v3/CHANGELOG.md [hunk 9]
```diff
@@ -8,6 +8,43 @@ nav_weight: 1000
...
+
+### Fixed
```
Comment (marksdk) [inline collections/_web/v3/CHANGELOG.md]:
```suggestion
### Fixed

```

File: collections/_web/v3/CHANGELOG.md [hunk 10]
```diff
@@ -8,6 +8,43 @@ nav_weight: 1000
...
+- Make sure selected/highlighted floor is always visible (no need to scroll) when focusing building.
```
Comment (marksdk) [inline collections/_web/v3/CHANGELOG.md]:
```suggestion
- Make sure selected/highlighted Floor is always visible (no need to scroll) when focusing Building.
```

**Pull request merged**